### ISSUE TITLE
UX - cq3 / operation - receiver.name_as_text() dumps column name as hex.

### ISSUE THREAD

**Issue by fee-mendes** [scylladb/scylladb#15882]
HEAD https://github.com/scylladb/scylladb/commit/949e9f1205aa397476f0303aa31891e152e7cf78

```cql
cqlsh:test> CREATE TABLE x (pk int PRIMARY KEY, a_fancy_name int, udt_fancy frozen<list<int>>);

cqlsh:test> UPDATE x SET udt_fancy = udt_fancy + [1] WHERE pk=0;
InvalidRequest: Error from server: code=2200 [Invalid query] message="Invalid operation (udt_fancy = udt_fancy + [1]) for frozen collection column 7564745f66616e6379"

cqlsh:test> UPDATE x SET a_fancy_name = a_fancy_name + 1 WHERE pk=0; 
InvalidRequest: Error from server: code=2200 [Invalid query] message="Invalid operation (a_fancy_name = a_fancy_name + 1) for non counter column 615f66616e63795f6e616d65"
```

Likely missing enclosing `to_string()` ?

**Comment by fee-mendes** [comment]
Probably in the realm of https://github.com/scylladb/scylladb/issues/1703

**Comment by alexander-tur** [comment]
@fee-mendes , I can't reproduce. Seems It's fixed.

**Comment by fee-mendes** [comment]
Indeed, it doesn't reproduce in nightly, though it did (and still does in 5.2.10). As it is fixed, then no need to chase whatever fixed it, this is cosmetics. Closed.

**Comment by nyh** [comment]
I fixed this in 4eabb3f429298f8e173e34ffb9827dd84ec39156, for issue #13657.  As usual, I found this bug with a translated Cassandra unit test ;-)

Issue #13657 is still listed as a backport candidate, we never decided not to backport it. @fee-mendes, what do you think?

**Comment by nyh** [comment]
@fee-mendes I think you didn't see this bug in the version you thought you did - my fix that I mentioned above reach Scylla 5.4, so indeed would have been broken in 5.2.10, but shouldn't have been broken on the specific version you mentioned (949e9f1205aa397476f0303aa31891e152e7cf78).

**Comment by fee-mendes** [comment]
> Issue https://github.com/scylladb/scylladb/issues/13657 is still listed as a backport candidate, we never decided not to backport it. @fee-mendes, what do you think?

Well, it is cosmetics. :) Yet, if it turns out to be effortless, then why not? ;-)
